### ISSUE TITLE
Can't search alphabetical words on the HTML quick search generated with language='ja'

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1340]
Search feature with language='ja' generate broken 'searchindex.js' file. Japanese words are OK, but alphabetical words prefixed with a whitespace. 

This issue reported by https://groups.google.com/d/msg/sphinx-users/1z4Y083Z-9o/fst4kXZAPkAJ

conf.py:

```
#!python

language='ja'
```

index.rst:

```
#!rst

It's a beautiful day.
```

searchindex.js:

```
#!js

Search.setIndex({envversion:42,terms:{" beautiful":0, ...}})
```

It caused by sphinx.search.ja.TinySegmenter.split() method.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1340
- Originally reported by: Takayuki Shimizukawa
- Originally created at: 2014-01-06T06:06:01.785

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-01-06 05:11:01+00:00_

quick fix. need test.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-01-06 23:51:25+00:00_

Fix: Can't search alphabetical words on the HTML quick search generated with language='ja'. Closes #1340

→ <<cset b17fea855b2e613fe7d879079caac7073ae7bb26>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:33:11+00:00_

Removing version: 1.1 (automated comment)
